docs/user/whats-new.asciidoc

```diff
@@ -11,4 +12,90 @@ coming[8.0.0]
 
 // tag::notable-highlights[]
 
+[float]
+=== Security features enabled by default
+
+Authentication, authorization, and multi-tenancy
+with {kib} Spaces are on-by-default, with almost no configuration effort.
+When you run {kib} for the first time, you'll be presented with a new interactive
+UI that will help you set up {kib} and connect it securely to {es}.
+Simply paste the enrollment token that you received when you ran
+{es}, and {kib} will set up all the previous security features
+and connect {es} over TLS.
+
+[role="screenshot"]
+image::images/highlights-security.png[Configure Elastic to get started]
+
+
+[float]
```

Comment:
```suggestion
[float]
[[index-pattern-rename]]
```
Perhaps we could add an anchor here? Then, when I link from the Cloud docs to explain the name change, the link will go right to this section.